### PR TITLE
Add prometheus metrics

### PR DESCRIPTION
**Pull request by chicobentojr**
### :sparkles: Enhancements

* Add Prometheus metrics to Send, Receive and Delete messages

### CODE REVIEW

**Review by felipemfp — APPROVED** [review]
LGTM :tada::+1:

File: collector.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+func NewSQSServiceCollector(opts *SQSServiceCollectorOpts) *SQSServiceCollector {
+	prefix := opts.Prefix
+	if prefix != "" && !strings.HasSuffix(opts.Prefix, "_") {
+		prefix += "_"
+	}
+	return &SQSServiceCollector{
+		messageCalls: prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: fmt.Sprintf("db_%smessage_calls", prefix),
```
Comment (jamillosantos) [inline collector.go]: I think `db_` is not a suitable prefix for these metrics. Update all of them.

File: service.go [hunk 2]
```diff
@@ -179,90 +185,368 @@ func (service *SQSService) RunWithSQS(handler func(client *sqs.SQS) error) error
...
+		}
+
+		service.Collector.messageTrafficAmount.With(metricLabels).Inc()
+		if input.MessageBody != nil {
+			service.Collector.messageTrafficSize.With(metricLabels).Add(float64(len(*input.MessageBody)))
+		}
+
+		return output, err
 	}
 	return nil, rscsrv.ErrServiceNotRunning
```
Comment (jamillosantos) [inline service.go:219]: Would not this be counted as an error? Same for others, you can reply if do not agree.

**Review by jamillosantos — CHANGES_REQUESTED** [review]

File: service.go (hunk 2 above)
Comment (chicobentojr) [reply service.go:219]: I was thinking that we can retrieve the number of `ErrServiceNotRunning`, without adding a new counter, calculating as: `messageCalls - (messageSuccess + messageFailures)`.
If we count ServiceNotRunning also as an error, we lost this possibility.

Also, I think that calling a method without starting service is a different kind of error and will be detected before.

File: collector.go (hunk 1 above)
Comment (chicobentojr) [reply collector.go]: Ok. Done :heavy_check_mark:

**Review by jamillosantos — CHANGES_REQUESTED** [review]
Sorry for not being clear man. When I was saying about the `db_` prefix, I really meant replacing it in favour of `sqs_` or something similar. Prefixes are your friend when dealing with metrics.

**Pull request merged**